go-controller/pkg/cni/helper_linux.go

```diff
@@ -338,6 +338,43 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 	return []*current.Interface{hostIface, contIface}, nil
 }
 
+func (pr *PodRequest) deletePodConntrack() {
+	if pr.CNIConf.PrevResult == nil {
+		return
+	}
+	result, err := current.NewResultFromResult(pr.CNIConf.PrevResult)
+	if err != nil {
+		klog.Warningf("could not convert result to current version: %v", err)
+		return
+	}
+
+	for _, ip := range result.IPs {
+		// Skip known non-sandbox interfaces
+		if ip.Interface != nil {
+			intIdx := *ip.Interface
+			if intIdx >= 0 &&
+				intIdx < len(result.Interfaces) && result.Interfaces[intIdx].Sandbox == "" {
+				continue
+			}
+		}
+		if ip.Version == "4" {
+			filterTypes := []netlink.ConntrackFilterType{netlink.ConntrackOrigSrcIP, netlink.ConntrackOrigDstIP, netlink.ConntrackReplyAnyIP}
```

Comment:
Since OVN distinguishes multiple networks using different conntrack zones, I think we need to filter based on the zone as well. In the future if we ever supported multiple networks, we could have overlapping IP ranges on different pods, and this could potentially remove an entry for another pod as well.

Reply:
This is going to be difficult.  The CT zones are per-port, and the CT zones in question here are actually the ones _not_ for the port that was just deleted.  Deciding which ones should be cleared and which ones shouldn't would also require knowing how the ports map to the logical network topology, and which might have entries for the IP you want to clear.